modules/configuring/proc-mounting-additional-files-in-your-custom-configuration-using-rhdh-operator.adoc

```diff
@@ -0,0 +1,76 @@
+[id="mounting-additional-files-in-your-custom-configuration-using-rhdh-operator"]
+= Mounting additional files in your custom configuration using the {product} operator
+
+You can use the {product-short} Operator to mount extra files, such as ConfigMaps and Secrets, to the container in a preferred location.
+
+The `mountPath` field specifies the location where a ConfigMap or Secret is mounted. The behavior of the mount, whether it includes or excludes a `subPath`, depends on the specification of the `key` or `mountPath` fields.
+
+* If `key` and `mountPath` are not specified: Each key or value is mounted as a `filename` or content with a `subPath`.
+* If `key` is specified with or without `mountPath`: The specified key or value is mounted with a `subPath`.
+* If only `mountPath` is specified: A directory containing all the keys or values is mounted without a `subPath`.
+
+[NOTE]
+====
+* {ocp-short} does not automatically update a volume mounted with subPath. By default, the {product-very-short} operator monitors these ConfigMaps or Secrets and refreshes the {product-very-short} Pod when changes occur.
+* For security purposes, {product} does not give the Operator Service Account read access to Secrets. As a result, mounting files from Secrets without specifying both mountPath and key is not supported.
+====
+
+
+.Prerequisites
+* You have developer permissions to access the {ocp-short} cluster containing your {product-short} instance using the {openshift-cli}.
+* link:{installing-on-ocp-book-url}[Your {ocp-short} administrator has installed the {product} Operator in {ocp-short}].
+
+.Procedure
+
+. In {ocp-short}, create your ConfigMap or Secret custom resource (CR) with the following YAML codes:
```

Comment:
Config maps and secrets are not custom resources, but standard Kubernetes objects.

Additionally, when referring to API object terms in general usage, use lowercase and separate multi-word API objects. 
* when referring to the resource name, write: config map, secret, custom resource
* When referring to the API object, write:  `ConfigMap` object, `Secret` object, `CustomResource` object. See https://redhat-documentation.github.io/supplementary-style-guide/#config-map and https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references


```suggestion
. In {ocp-short}, create your config map or secret with the following YAML:
```

Additionally